app/helpers/admin/republishing_helper.rb

```diff
@@ -39,6 +39,13 @@ def bulk_content_type_metadata
         confirmation_path: admin_bulk_republishing_confirm_path("all-documents-with-publicly-visible-editions-with-html-attachments"),
```

Comment:
Another commit message thing, if you could just add a line about why this is desirable please